packages/devtools_extensions/lib/src/template/extension_manager.dart

```diff
@@ -185,6 +196,33 @@ class ExtensionManager {
     }
   }
 
+  Future<void> _connectToDtd(String? dtdUri) async {
+    // TODO(kenz): investigate. this is weird but `dtdUri` != null even
```

Comment:
It sounds like the query param is actually being set somewhere when `dtdUri == null`, so it's being set with the string 'null'.